src/styles/_typography.scss

```diff
@@ -1,3 +1,15 @@
 h1 {
```

Comment:
In a file called `typography` usually called basic font styles for elements. You did it in `normalize`. Better to extend your header styles in file `normalize` (not here)